packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1383,6 +1383,11 @@ const schema: SchemaType<"Users"> = {
     label: "Posts added to tags I'm subscribed to",
     ...notificationTypeSettingsField(),
   },
+  notificationSubscribedSequencePost: {
+    label: "Posts added to sequences I'm subscribed to",
+    ...notificationTypeSettingsField({ channel: "both" }),
+    hidden: !allowSubscribeToSequencePosts
+  },
```

Comment:
I agree this is the idiomatic way to do this, but I do sure wish there was broader umbrella we could use.